sssom/cli.py

```diff
@@ -89,6 +89,28 @@
     help="Fields.",
 )
 
+predicate_filter_option = click.option(
+    "-F",
+    "--mapping-predicate-filter",
+    multiple=True,
```

Comment:
What does multiple do? Will it parse a space separated string into a list?

Reply:
Yes